src/util/Ui/CommandButtonProps.ts

```diff
@@ -128,26 +130,29 @@ export const getAddElementButtonProps = (editor: YaptideEditor): GroupedCommandB
 
 	const filtersTuple: CommandButtonTuple[] = [
 		[
-			'Custom filter',
+			'Particle filter',
```

Comment:
Particle filter support was addend in converter, however, this RP contains an old version of converter.

You can try rebasing this branch on master.
I checked it locally and there were nop merge conflicts.